CardData.py

```diff
@@ -112,15 +103,9 @@ def create_card_json(gwent_data_helper, patch):
         card['categories'] = []
         card['categoryIds'] = []
         categoriesSum = int(template.find('Categories').find('e0').attrib['V']);
-        # XML Card category is the sum of all categories of the card
-        for category in range(LAST_CATEGORY, 1, -1):
-            categoryPower = 2**category
-            if categoriesSum - categoryPower >= 0:
-                categoriesSum -= categoryPower
-                if category in CATEGORIES:
-                    card['categoryIds'].append("card_category_" + str(category))
-                if categoriesSum == 0:
-                    break
+        for category, bit in enumerate("{0:b}".format(categoriesSum)[::-1]):
```

Comment:
You could still get ride of CATEGORIES in the end if you let the en_us translation as the reference of non-existent categories.
```
categoryIds = []
for category, bit in enumerate("{0:b}".format(categoriesSum)[::-1]):
    if bit == '1':
        categoryIds.append("card_category_{0}".format(category))
categories_en_us = gwent_data_helper.categories["en-US"]
for category_id in categoryIds:
    if category_id in categories_en_us:
        card['categories'].append(categories_en_us[category_id])
        card['categoryIds'].append(category_id)
```

Reply:
Is it not possible that es-ES has it, but en-US does not?
(note: if you nest a `for ... in` use a `set`, e.g. `category_set = set(categories_en_us)`)